applications/asset_tracker_v2/doc/cloud_wrapper.rst

```diff
@@ -0,0 +1,162 @@
+|                              +----------------------------------------------------+           |
+|                              | $aws/things/<imei>/shadow/get/accepted/desired/cfg |           |
++------------------------------+----------------------------------------------------+-----------+
+
+Azure IoT Hub topics
+====================
+
+For simplicity, the following table omits certain metavalues present in topics and property bags used in Azure IoT Hub.
+For more information regarding MQTT topics and property bags in Azure IoT Hub, refer to the following documentation `Azure IoT Hub MQTT protocol support`_.
+
++------------------------------+-----------------------------------------+--------------+-----------+
+|               Data           |             Topic                       | Property bag | Direction |
++==============================+=========================================+==============+===========+
+| A-GPS requests               | devices/<imei>/messages/events/         | agps=get     | D2C       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| P-GPS requests               | devices/<imei>/messages/events/         | pgps=get     | D2C       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| Neighbor cell measurements   | devices/<imei>/messages/events/         | ncellmeas    | D2C       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| Button presses               | devices/<imei>/messages/events/         | NA           | D2C       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| Sensor/device data           | $iothub/twin/PATCH/properties/reported/ | NA           | D2C       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| Device configuration         | $iothub/twin/PATCH/properties/reported/ | NA           | D2C       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| Buffered sensor/device data  | devices/<imei>/messages/events/         | batch        | D2C       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| A-GPS response               | devices/<imei>/messages/devicebound/    | agps=result  | CD2       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| P-GPS response               | devices/<imei>/messages/devicebound/    | pgps=result  | C2D       |
++------------------------------+-----------------------------------------+--------------+-----------+
+| Device configuration updates | $iothub/twin/res/<code>/                | NA           | C2D       |
++------------------------------+-----------------------------------------+--------------+-----------+
```

Comment:
**suggestion**: to improve readability, split table in two, one for D2C and one for C2D, remove Direction column.